### PR TITLE
Roby dev

### PR DESCRIPTION
**Pull request by roby-handinhand**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DATABASE_URL=postgres://user:pwd@localhost:5433/postgres
+GOOGLE_CALLBACK_URL=http://localhost:5000/auth/google/callback
+GOOGLE_CLIENT_SECRET=your-client-secret
+GOOGLE_CLIENT_ID=your-client-id
+FACEBOOK_CLIENT_SECRET=your-client-secret
+FACEBOOK_CLIENT_ID=your-client-id
+FACEBOOK_CALLBACK_URL=http://localhost:5000/auth/facebook/callback
```
Comment (gik0geck0) [inline .env.sample:7]: I noticed that the Heroku configs have the callback URL with HTTP. Do they accept HTTP as a callback? SFDC connected apps force HTTPS.

File: views/index.ejs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	</style>
+</head>
+<body>
+<div class="container">
+
+	<div class="jumbotron text-center">
+		<p>Login or Register with:</p>
+
+		<a href="/login" class="btn btn-default"><span class="fa fa-user"></span> Local Login</a>
+		<a href="/signup" class="btn btn-default"><span class="fa fa-user"></span> Local Signup</a>
```
Comment (gik0geck0) [inline views/index.ejs:18]: I don't think we want a Local Login or Local Signup. OAuth only.

File: app/models/user.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
+// file app/models/user.js
+// define the model for User 
+
+
+// load the things we need
+
+var bcrypt   = require('bcrypt-nodejs');
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('user', {
```
Comment (gik0geck0) [inline app/models/user.js:10]: A userId as our primary key on a user would be good

**Review by gik0geck0 — APPROVED** [review]

File: .env.sample (hunk 1 above)
Comment (roby-handinhand) [reply .env.sample:7]: I'm using http when testing local, and https on the actual deployed heroku app callback.

File: views/index.ejs (hunk 2 above)
Comment (roby-handinhand) [reply views/index.ejs:18]: I'm rethinking that, given that Heroku Connect stores CRM data in the local postgres db, I'm thinking that, security-wise, we might be ok storing users local login data in the db as well.

File: app/models/user.js (hunk 3 above)
Comment (roby-handinhand) [reply app/models/user.js:10]: Lets discuss - I was assuming email is our unique key.

**Pull request merged**